bot/exts/christmas/hanukkah_embed.py

```diff
@@ -55,25 +56,26 @@ async def hanukkah_festival(self, ctx: commands.Context) -> None:
         month = str(today.month)
         year = str(today.year)
         embed = Embed()
-        embed.title = 'Hanukkah'
+        embed.title = "Hanukkah"
         embed.colour = Colours.blue
         if day in self.hanukkah_days and month in self.hanukkah_months and year in self.hanukkah_years:
             if int(day) == hanukkah_start_day:
                 now = datetime.datetime.utcnow()
-                now = str(now)
-                hours = int(now[11:13]) + 4  # using only hours
+                hours = now.hour + 4  # using only hours
                 hanukkah_start_hour = 18
                 if hours < hanukkah_start_hour:
                     embed.description = (f"Hanukkah hasnt started yet, "
                                          f"it will start in about {hanukkah_start_hour-hours} hour/s.")
-                    return await ctx.send(embed=embed)
+                    await ctx.send(embed=embed)
+                    return
                 elif hours > hanukkah_start_hour:
-                    embed.description = (f'It is the starting day of Hanukkah ! '
-                                         f'Its been {hours-hanukkah_start_hour} hours hanukkah started !')
-                    return await ctx.send(embed=embed)
+                    embed.description = (f"It is the starting day of Hanukkah ! "
+                                         f"Its been {hours-hanukkah_start_hour} hours hanukkah started !")
+                    await ctx.send(embed=embed)
+                    return
             festival_day = self.hanukkah_days.index(day)
-            number_suffixes = ['st', 'nd', 'rd', 'th']
-            suffix = ''
+            number_suffixes = ["st", "nd", "rd", "th"]
+            suffix = ""
             if int(festival_day) == 1:
```

Comment:
I realize this isn't in your PR, but doesn't `list.index` already return an `int`? Might as well remove this casting to int here.

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/dd20f59b9fb12288f286d479b5719697ab1a30f4.